src/CachePeer.h

```diff
@@ -143,7 +143,11 @@ class CachePeer
     char *digest_url = nullptr;
 #endif
 
-    int tcp_up = 0;         /* 0 if a connect() fails */
+    /// The number of failures sufficient to stop selecting this cache_peer. All
+    /// peer selection algorithms skip peers with zero tcp_up values. The
```

Comment:
Just for consistency:
```suggestion
    /// peer selection algorithms skip cache_peers with zero tcp_up values. The
```
or use 'peer' in all three lines.

Reply:
Seems to me all three should be "cache_peer". While the latter two could be interpreted as plural they are used as one piece of a compound noun.

Reply:
Commit f8de58d changed all "peer(s)" to "cache_peer(s)". I hope we do not need to argue about pieces of compound nouns in this context!